File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        r=0
        s= set(nums)
         
        for num in nums:
            streak = 0
            curr = num
            while curr in s:
                streak += 1
                curr += 1
            r = max(r,streak)
        return r

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer, Integer> freq = new HashMap<>();

        for (int num: nums){
            freq.put(num, freq.getOrDefault(num,0)+1);
        }
        List<Integer> list = new ArrayList<>(freq.keySet());
        list.sort((a,b) -> freq.get(b) - freq.get(a)); 

        int[] result = new int[k];
        for (int i=0; i<k; i++){
            result[i] = list.get(i);
        }
        return result;
    }
}
